src/renderer/components/TokenTips.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import React from "react";
+import styled from "styled-components";
+import { colors } from "~/renderer/styles/theme";
+import Box from "~/renderer/components/Box";
+import Text from "~/renderer/components/Text";
+import InfoCircle from "~/renderer/icons/InfoCircle";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+import { Trans } from "react-i18next";
+import { shell } from "electron";
+
+const TokenTipsContainer: ThemedComponent<{}> = styled(Box)`
+  background: ${colors.pillActiveBackground};
+  color: ${colors.wallet};
+  font-weight: 400;
+  padding: 16px;
+`;
+
+const Strong: ThemedComponent<{}> = styled.span`
+  font-weight: 700;
+`;
+const LearnMore: ThemedComponent<{}> = styled.a`
```

Comment:
avoid using href links to redirect to external pages. prefer the use of a styled button instead

Reply:
ok